templates/tapestry.js

```diff
@@ -1218,15 +1218,15 @@ function tapestryTool(config){
             .append("svg:foreignObject")
             .html(function (d) {
                 return '<i id="mediaButtonIcon' + d.id + '"' + 
-                    ' class="' + getIconClass(d.mediaType, 'play') + ' mediaButtonIcon"' +
+                    ' class="' + ' mediaButtonIcon"' +
```

Comment:
This doesn't work. We only want the icon to change for text nodes (i.e. nodes that have their node type set as "Text"). You'll need to keep this function and instead change the implementation of the function and maybe add some CSS.